### DISCUSSION TITLE
Lancement

### DISCUSSION THREAD

**Discussion by sjinko** [CUPUM/nplex#179] [General]
Discussions sur le lancement de nplex (projet : https://github.com/orgs/CUPUM/projects/98) et des epics et tasks qui le composent.
@frroye @SarahTannir @roxanekasprzyk 

Voici l‘organisation proposée: 
@ffroye est responsable du [projet de lancement](https://github.com/orgs/CUPUM/projects/98), c’est à dire de s’assurer que les issue Epics soient fait dans les temps
@roxanekasprzyk est responsable du issue #175 
@SarahTannir est responsable du issue #176 et #177 
@frroye est responsable du issue #178

Chacune doit donc préciser les tâches (tasks) à accomplir dans leur Epic et les attribuer à quelqu’un. Toujours n’avoir qu’une seule personne à la fois associée à un issue.

Vous pouvez écrire ici pour les questions / remarques générales qui ne relèvent pas d’un issue.

**Comment by roxanekasprzyk** [comment]
Bonjour @iolyd, 

Je viens de recevoir par email de nouveaux projets à intégrer à la plateforme de la part des firmes qui n'avaient pas eu le temps de le faire avant le 10 octobre. Peut-on les enregistrer à la plateforme ? 
Je les enregistre dans le [drive](https://drive.google.com/drive/folders/1fBwyzJhBOHCZ2xvCx-TsT4KfXSZwbe31?fbclid=IwAR1kiUWPqLDnvVkjJHnAgPaMCxmTsDh4Hvf5XKt_g2EPGsOgaZs--rikhd8)

Merci à toi, 

Roxane

**Reply by emmnull** [reply]
Oui, les nouveaux projets peuvent être ajoutés à la plateforme. Il y a quelques fonctionnalités pour l'instant encore instables et pour lesquelles je vise déployer de nouvelles versions d'ici mercredi prochain (le 1er novembre), mais ça ne devrait pas prévenir l'ajout de nouvelles fiches. Cependant, je proposerais d'attendre encore quelques jours avant de téléverser les **images** des projets car le système présent dans la version en ligne est désuet et sera remplacé lors du prochain déploiement.
